### PR TITLE
PXC-4023 8.0.29-21 release notes and release tasks (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
A  docs/release-notes/8.0.29-21.md
M  docs/release-notes/release-notes_index.md
M  mkdocs-base.yml

### CODE REVIEW

File: docs/release-notes/8.0.29-21.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+!!! note
+
+    Percona Server for MySQL has changed the default for the [supported DDL column operations](https://dev.mysql.com/doc/refman/8.0/en/innodb-online-ddl-operations.html) to ALGORITHM=INPLACE. This change fixes the corruption issue with the INSTANT ADD/DROP COLUMNS (find more details in [PS-8292](https://jira.percona.com/browse/PS-8292).
+
+    In MySQL 8.0.29, the default setting for supported DDL operations is ALGORITHM=INSTANT. You can explicitly specify ALGORITHM=INSTANT in DDL column operations.
+
+## Bugs Fixed
+
+* [PXC-3982](https://jira.percona.com/browse/PXC-3982): A replica node is also an asynchronous slave. A SQL thread failed at started if this node was not ready to accept connections while joining the cluster.
```
Comment (kamil-holubicki) [inline docs/release-notes/8.0.29-21.md]: Maybe this way? : When the replica node is also an asynchronous slave, a SQL thread failed at the start if this node ...

Comment (patrickbirch) [reply docs/release-notes/8.0.29-21.md]: Rewritten.

**Review by kamil-holubicki — APPROVED** [review]
lgtm

**Review by george-lorch — APPROVED** [review]
lgtm

**Pull request merged**